README.md

```diff
@@ -1,62 +1,65 @@
+This is the primary communication and governance hub of the Node.js
+Foundation's Technical Steering Committee. Its goals are:
 
-For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md) adopted by the Node.js Foundation Board of Directors on June 17th 2015.
+* Maintaining, releasing and improving a healthy Node.js Core project.
+* Fostering and encouraging a healthy Node.js open source ecosystem.
 
-If your project is interested in joining the Node.js Foundation please read the [Project Lifecycle.md](./Project-Lifecycle.md) documentation.
+# TSC
 
-## TSC Members
-
-TSC members are responsible for top level technical community concerns. The role is 
-mostly administrative and is responsible for admitting new Top Level Projects, Top Level
-Working Groups, and advocating for any needs in the technical side of the foundation to
-the Node.js Foundation Board of Directors.
+The TSC has many working groups and projects. Each group and project
+has a connection to the development of Node.js Core but their work and
+responsibilities often extend far beyond Core and potentially even Node.js.
 
-TSC members can nominate new members at any time. Candidates for membership tend to be people
-who have a competancy for community management and a high tolerance and patience for process
-minutiae as the TSC delegates most of its responsibilities to other projects and working groups.
+The TSC also leads several efforts to improve the health and sustainability
+of the ecosystem but it is not a long term host of ecosystem projects which are
+not integrated into Core development.
 
-Every Top Level Project not currently incubating can appoint someone to the TSC who they elect
-at their own discretion.
+Each project and working group is autonomous. The scope of work and 
+autonomy is described in their charter. The TSC works to increase communication 
+and collaboration between all these parts.
 
-## Top-Level WGs and TLPs
+Adding a new project or working group can be done through a pull request. Since
+each group is autonomous any new group taking on responsibility from another
+group would need that group's approval.
 
-* [Working Groups](WORKING_GROUPS.md)
-* Mentors
+## Projects
 
-    Project mentorship is not a technical role. In fact, mentors are
-    discouraged from giving technical advice to projects. Instead, the
-    purpose of mentorship is to encourage and improve a projects ability
-    to be participatory, transparent, and effective. Mentors are there to
-    help projects adopt and iterate on policies and processes that achieve
-    these goals and eventually allow them to graduate the incubation phase.
+* Core
+* Website
+* Streams
+* Docker
+* nan
 
-      * Mikeal Rogers (@mikeal)
-* Top-Level Projects
- * Core TLP
-  * Core WGs (streams, http, Intl)
+## Working Groups
 
-## Policy Change Proposal Process
+* Inclusivity
```

Comment:
Hrm... this is tricky. We should probably move the WORKING_GROUPS.md from nodejs/node here and commit it as is in this PR and then open another one over it for the re-scoping so that we can better track the changes. That should work fine though, there's no point in writing new charters if this PR doesn't pass.